usmqe/api/etcdapi/etcdapi.py

```diff
@@ -7,217 +7,28 @@
 import time
 import requests
 import pytest
+import usmqe.api.api as api
```

Comment:
I think we should change dir structure a little bit. Because yo uuse here 
```usmqe.api.api``` and later ```api.Api```